packages/pyo3/pyproject.toml

```diff
@@ -1,3 +1,25 @@
+[project]
+name = "graspologic-native"
+maintainer = "Dax Pryce"
+maintainer-email = "daxpryce@microsoft.com"
+requires-python = ">=3.6,<3.12"
```

Comment:
If you want to, we can try to extend this to 3.12 too. It's currently in final so we can probably extend this and see how it works (the build matrix in build.yaml will need updated too to include 3.12). If it fails just back it out, but we should try it while we're in here :)